andela/pippin-ah-backend
# [Feature: #162727551] Send email notification to users when they register

#### What does this PR do?
- Send email notification to users when they register

#### Description of Task to be completed?
- [x] Installs @sendgrid/mail from sendgrid
- [x] Configure signup controller to send an Email to user on successful registration
- [x] Activate user account when they click link.
- [x] Write one test.

#### How should this be manually tested?
- Fire up Postman
- Url: `http://localhost:3000/api/v1/users`
- Enter Registration details 
     - `Valid email`
     - `Valid Username`
     - `Valid password`
- Click Send
- You should receive An email from learnGround welcoming you.

#### What are the relevant pivotal tracker stories?
- [#162727551](https://www.pivotaltracker.com/story/show/162727551)
